File: sdkwork-app-sdk-go/types/partner_vo.go
package types

// 合作伙伴响应
type PartnerVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	PartnerId string `json:"partnerId"`
	Name string `json:"name"`
	Level string `json:"level"`
	Status string `json:"status"`
	PromotionCode string `json:"promotionCode"`
	ContactName string `json:"contactName"`
	ContactPhone string `json:"contactPhone"`
	TotalCommission float64 `json:"totalCommission"`
	WithdrawableCommission float64 `json:"withdrawableCommission"`
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/partner_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PartnerVO:
    """合作伙伴响应"""
    created_at: str = None
    updated_at: str = None
    partner_id: str = None
    name: str = None
    level: str = None
    status: str = None
    promotion_code: str = None
    contact_name: str = None
    contact_phone: str = None
    total_commission: float = None
    withdrawable_commission: float = None
